contrib/scripts/codepage_to_utf8.py

```diff
@@ -0,0 +1,36 @@
+#!/bin/python3
+"""
+Convert file from legacy codepage to Unicode UTF-8
+"""
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser(description="Convert file from legacy codepage to Unicode UTF-8",
+                                 formatter_class=argparse.RawTextHelpFormatter)
+parser.add_argument("filename", help="filename to convert")
+parser.add_argument("--cp", type=int, default=437, help="codepage to convert from (default: 437)")
+parser.add_argument("-o", "--output", type=str,
+                    help="output filename (default: overwrite input file)")
+args = parser.parse_args()
+
+path = os.path.dirname(__file__)
+
+if not args.output:
+    args.output = args.filename
+
+try:
+    with open(os.path.join
+                  (path, './' + args.filename), 'r', encoding='cp' + str(args.cp)) as input_file:
+        lines = input_file.read()
+        with open(os.path.join(path, './' + args.output), 'w', encoding='utf-8') as output_file:
+            output_file.write(lines)
+            output_file.close()
```

Comment:
Correct me if I'm wrong with my limited Python knowledge.  It seems you're reading the entire file into memory with `lines = input_file.read()`?  That's fine, and probably needed if you want to overwrite the existing file.

I would suggest closing the input file before writing the output.  I believe files get automatically closed at the end of the `with` block.  If my assumptions are correct:

1. Remove a layer of indention from the 2nd `width` block.
2. Remove `output_file.close()` as that should be handled automatically.

Reply:
Yeah, it gets closed automatically, the close() is indeed not strictly needed.

But the indentation from the 2nd with block is required.
The first 'with' line had to be broken into two lines it exceeded 100 characters, which caused the linter to fail.

Reply:
Significant white-space problems... :roll_eyes: 

I was just worried about potential issues caused by holding open a file handle while modifying the same file under a different handle.

Reply:
Not sure what those significant whitespace problems are. An a empty line here and there to separate code blocks? If you are talking indentation, that is required by Python.

Reply:
I was just poking fun at Python for having significant whitespace and it apparently causing problems with the linter.  You can ignore that comment.

I see you've made some changes so now the first `open` fits on one line.  Can you just remove the indention from the second `with` block that outputs (putting on the same level on indentation as the input block)?  That way it happens after the input file is closed.  That's all I'm asking.